dataland-frontend/tests/e2e/specs/vsme/RequestingAndGrantingAccess.ts

```diff
@@ -0,0 +1,229 @@
+import { describeIf } from '@e2e/support/TestUtility';
+import { admin_name, admin_pw, getBaseUrl, reader_name, reader_pw } from '@e2e/utils/Cypress';
+import { getKeycloakToken } from '@e2e/utils/Auth';
+import { DataTypeEnum, type StoredCompany, type VsmeData } from '@clients/backend';
+import { generateDummyCompanyInformation, uploadCompanyViaApi } from '@e2e/utils/CompanyUpload';
+import { uploadVsmeFrameworkData } from '@e2e/utils/FrameworkUpload';
+import { type FixtureData } from '@sharedUtils/Fixtures';
+
+describeIf(
+  'As a user, I expect to request access to private datasets or grant or decline access to my private datasets ',
```

Comment:
CI ist aktuell kaputt und in der CD gibt es auch einen Fehler:
![grafik](https://github.com/user-attachments/assets/460f41de-3c3d-4e65-b680-fcd69776db99)

Reply:
fixed.  lag an einem fehlenden watcher